src/icons/Exclamation.tsx

```diff
@@ -0,0 +1,45 @@
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
+
+const svgElementClasses = classnames(
+  stroke('stroke-formal-accent'),
+  strokeWidth('stroke-1')
+)
+
+export default function () {
+  return (
+    <svg
+      width="13"
+      height="13"
+      viewBox="0 0 13 13"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <circle
+        cx="6.50004"
```

Comment:
We can round those numbers I think, try doing `cx="6.5" cy="6.5"` etc

Reply:
Rounded them. Thanks